app/src/TxManager.ts

```diff
@@ -24,34 +21,41 @@ type LiquidationTxInfo = {
     retries: number;
 }
 
+const MAX_GAS_PRICE_FOR_CHAIN: Map<number, string> = new Map<number, string>([
+    [1, "100000000000"],
+    [5, "10000000000"],
+    [10, "10000000"],
+    [42161, "1000000000"],
```

Comment:
These would be more readable if you used web3.js conversion tools (from gwei --> wei)